test/resources/proposals/test-many-proposals.js

```diff
@@ -0,0 +1,51 @@
+module.exports = [
+  // updateThresholdNewAuction(uint _thresholdNewAuction)
+  {
+    type: 'CHANGE-THRESHOLD-NEW-AUCTION',
+    newThresholdInUsd: 2000
```

Comment:
`uint` in the commented function signature suggests to me that `newThresholdInUsd` is in uint already, but later you do `toWei` in the script.
Maybe add another comment to clarify

Reply:
Yup, the `updateThresholdNewAuction(uint _thresholdNewAuction)` is just a copy & paste of the signature, just to make it simpler to know exactly which operation will be invoked in the SC. 

At the beginning i called the param `newThreshold`, I added `newThresholdInUsd` to show that the units was USD (not wei of $). Anyways, I'll add a comment so it's more clear.